src/elements/content-sidebar/versions/VersionsSidebarContainer.js

```diff
@@ -43,6 +47,7 @@ type Props = {
 type State = {
     error?: MessageDescriptor,
     isLoading: boolean,
+    isArchiveFile: boolean,
```

Comment:
Minor, but we should alphabetize props and object properties when possible.

Reply:
Updated